### PR TITLE
RoCC PTW refactoring

### PR DESCRIPTION
**Pull request by yunsup**
@aswaterman @zhemao cleaning up the RoCC PTW ports.

### CODE REVIEW

File: src/main/scala/Configs.scala [hunk 1]
```diff
@@ -153,6 +153,7 @@ class DefaultConfig extends Config (
       }
       case BuildRoCC => Nil
       case RoccNMemChannels => site(BuildRoCC).map(_.nMemChannels).foldLeft(0)(_ + _)
+      case RoccNPTWPorts => site(BuildRoCC).map(_.nPTWPorts).foldLeft(0)(_ + _)
```
Comment (aswaterman) [inline src/main/scala/Configs.scala]: What's with the inconsistent capitalization between RoCC and Rocc?

Comment (yunsup) [inline src/main/scala/Configs.scala]: I was trying to be consistent with other parameters around: RoccNMemChannels, RoccNCSRs.

Comment (colinschmidt) [inline src/main/scala/Configs.scala]: At some point in the past I think the parameters were changed to match some Scala style (camel case). I personally like the capital Cs but either way we should pick one

Comment (aswaterman) [inline src/main/scala/Configs.scala]: It seems that Java and Scala favor using consistent casing for acronyms, even in camelcase (e.g. package xml, class XMLParser.  So RoCC is fine.

Comment (yunsup) [inline src/main/scala/Configs.scala]: There should be another commit which changes all Rocc references to RoCC, so I'm submitting this PR again only with the updated rocket hash tag.
